apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -116,6 +121,11 @@ function unload_core_fse() {
 	remove_action( 'admin_menu', 'gutenberg_site_editor_menu', 9 );
 	remove_action( 'admin_menu', 'gutenberg_remove_legacy_pages' );
 	remove_action( 'admin_bar_menu', 'gutenberg_adminbar_items', 50 );
+	remove_action( 'admin_menu', __NAMESPACE__ . '\hide_nav_menus_submenu' );
+
+	// Shows the AMP menu.
+	add_action( 'admin_menu', 'amp_add_customizer_link' );
```

Comment:
We should probably check if this function exists before adding an action for it.

cc @mrfoxtalbot - may be the cause of the issue you ran into.